functional-samples/cookbook.wasm-helloworld-print/manifest.json

```diff
@@ -2,7 +2,7 @@
   "name": "WASM Load Example - Helloworld",
   "version": "1.0",
   "manifest_version": 3,
-  "description": "Inspect background service worker console for output",
+  "description": "This recipe shows how to use WASM in Manifest V3.",
```

Comment:
```suggestion
  "description": "Shows how to use Web Assembly (WASM) in Manifest V3.",
```

Reply:
What's the difference between this and the previous item? Please make the descriptions answer that question.

Reply:
Done. https://github.com/GoogleChrome/chrome-extensions-samples/pull/994/commits/6e115efd450f9e5fba63fa4d94c3e64a81da80a6